### PR TITLE
Boros to be able to pick lanni kingsguard

### PR DESCRIPTION
**Pull request by adkateki**
Not sure if this is the way to use CardMatcher, just proposing in case it can save some time in order to fix Boros.

### CODE REVIEW

File: server/game/cards/19-JS/SerBorosBlount.js [hunk 1]
```diff
@@ -11,7 +11,7 @@ class SerBorosBlount extends DrawCard {
             gameAction: GameActions.search({
                 title: 'Select a character',
                 topCards: 10,
-                match: { type: 'character', trait: ['Kingsguard', 'Knight'], not: { faction: 'lannister' } },
+                match: { type: 'character', or: { trait: ['Knight'], not: { faction: 'lannister' } }, or: { trait: ['Kingsguard'] } },
```
Comment (Patane97) [inline server/game/cards/19-JS/SerBorosBlount.js:14]: Close; the `or` is relatively new to the matcher, and simply takes in an array of "or matches" to match. For example, the above should look like this:
```suggestion
                match: { type: 'character', or: [{ trait: ['Knight'], not: { faction: 'lannister' } }, {trait: 'Kingsguard'}] },
```
The above `or` is an array containing the element `{ trait: ['Knight'], not: { faction: 'lannister' } }` and the element `{trait: 'Kingsguard'}`

Happy to approve once adjusted :)

**Review by Patane97 — CHANGES_REQUESTED** [review]
Appreciate the quick suggested fix! Unfortunately didn't pass our unit tests, but I've commented below where the adjustment should go. :)

File: server/game/cards/19-JS/SerBorosBlount.js (hunk 1 above)
Comment (adkateki) [reply server/game/cards/19-JS/SerBorosBlount.js]: Hi! Just updated it!

**Review by Patane97 — APPROVED** [review]
Looks good now!

**Pull request merged**